Automattic/newspack-theme
# fix: update single post footer print styles

### All Submissions:


### Changes proposed in this Pull Request:

This PR cleans up some weirdness in the single post print styles:

* If turned on, the previous/next links are visible.
* If turned on, the author bio is visible -- though this isn't always illogical, if you have the author bio truncated, or any of the social links turned on, it doesn't work for the print version of the post. It's also not entirely post content, so I opted to hide it. 
* If you have the above footer widget enabled, it prints.

Closes #1785

### How to test the changes in this Pull Request:

1. Start with a single post with a visible author bio.
2. Navigate to Customizer > Template Settings > Post Settings, and turn on the previous/next links.
3. Navigate to Customizer > Widgets > Above Footer widget, and add a widget.
4. Preview the print version of the page and note that all of the above is visible.
5. Apply the PR and run `npm run build`.
6. Preview the print version and confirm the previous/next links, author bio and above footer links no longer display.

### Other information:
